rewrite-maven/src/main/java/org/openrewrite/maven/MavenSettings.java

```diff
@@ -150,12 +150,6 @@ void maybeDecryptPasswords(ExecutionContext ctx) {
         }
     }
 
-
-    public static boolean readFromDiskEnabled() {
-        final String propertyValue = System.getProperty("org.openrewrite.test.readMavenSettingsFromDisk");
-        return propertyValue != null && !"false".equalsIgnoreCase(propertyValue);
-    }
```

Comment:
Apologies; on second thought best to keep this one (maybe deprecated), because it's parent class loaded as well.